src/apps/investments/views/admin/client/InvestmentProjectAdmin.jsx

```diff
@@ -26,70 +21,55 @@ const InvestmentProjectAdmin = ({
   projectName,
   projectStage,
   stages,
-  stageUpdated,
 }) => {
-  useEffect(() => {
-    if (stageUpdated) {
-      addMessage('success', 'Project stage saved')
-      window.location.href = urls.investments.projects.project(projectId)
-    }
-  }, [stageUpdated])
   const newStageOptions = stages.filter(
     (stage) => stage.value != projectStage.id
   )
   return (
     <Task>
-      {(getTask) => {
-        const updateStageTask = getTask(TASK_UPDATE_STAGE, STATE_ID)
-
-        return (
-          <>
-            <LocalHeader
-              heading={'Change the project stage'}
-              breadcrumbs={[
-                { link: urls.dashboard(), text: 'Home' },
-                { link: urls.investments.index(), text: 'Investments' },
-                { link: urls.investments.projects.index(), text: 'Projects' },
-                {
-                  link: urls.investments.projects.project(projectId),
-                  text: projectName,
-                },
-                { text: 'Admin' },
-              ]}
+      <>
+        <LocalHeader
+          heading={'Change the project stage'}
+          breadcrumbs={[
+            { link: urls.dashboard(), text: 'Home' },
+            { link: urls.investments.index(), text: 'Investments' },
+            { link: urls.investments.projects.index(), text: 'Projects' },
+            {
+              link: urls.investments.projects.project(projectId),
+              text: projectName,
+            },
+            { text: 'Admin' },
+          ]}
+        />
+        <Main>
+          <H4 as="h2">Project details</H4>
+          <InsetText>
+            <p>Project name: {projectName}</p>
+            <StyledP>Current stage: {projectStage.name}</StyledP>
+          </InsetText>
+          <TaskForm
+            actionLinks={[
+              {
+                children: 'Save',
+                href: urls.investments.projects.project(projectId),
+              },
+            ]}
+            analyticsFormName="investment-project-admin"
```

Comment:
```suggestion
            analyticsFormName="investmentProjectAdmin"
```

The performance analysts have asked us to have this in snake case